wordpress-mobile/WordPress-iOS
# Integrate aztec beta9

Integrates Aztec pod beta 9 on the app.

To test:
 - Activate the new editor
 - Give it a spin to see if there is any major issues.
 - Test copy paste of block of text inside the editor, because this was one of the areas that was causing us trouble.

Needs review: @jleandroperez  and @astralbodies please give it spin around!
